sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -258,6 +417,17 @@ class Analyzer(override val catalogManager: CatalogManager)
     TypeCoercion.typeCoercionRules
   }
 
+  private def resolveExpressionByPlanOutput(
```

Comment:
seems we can just `import Analyzer._` in `class Analyzer`

Reply:
I alias in the class because the public method in Object needs a `resolver` as parameter.